Dockerfile

```diff
@@ -17,4 +17,7 @@ ENV MAVEN_CONFIG "$USER_HOME_DIR/.m2"
 COPY settings.xml $USER_HOME_DIR/.m2/
 
 COPY ./build.sh /project/build
+COPY ./docker-entrypoint.sh /
+
+ENTRYPOINT ["/docker-entrypoint.sh"]
```

Comment:
why don't you use `ENTRYPOINT ["/project/build", "/project/source"]` instead of using a `docker-entrypoint.sh` script? Is it not equivalent?